test/Operators/hybrid/dss_opt.jl

```diff
@@ -0,0 +1,41 @@
+using Test
```

Comment:
@simonbyrne, can we include this in `test/runtests.jl`?

Reply:
ah, i meant to